pr_agent/algo/ai_handlers/litellm_ai_handler.py

```diff
@@ -126,6 +142,7 @@ async def chat_completion(self, model: str, system: str, user: str, temperature:
                 "deployment_id": deployment_id,
                 "messages": messages,
                 "temperature": temperature,
+                "max_tokens": get_settings().config.max_model_tokens,
```

Comment:
p.s.
this is wrong, and very dangerous.

if I understand correctly, litellm `max_tokens` is for the **output** `config.max_model_token` is something vet different

Reply:
fixed